docs/quickstarts/packet.md

```diff
@@ -116,6 +116,8 @@ cluster "packet" {
   facility = "ams1"
   project_id = "89273817-4f44-4b41-9f0c-cb00bf538542"
 
+  controller_type = "c3.small.x86"
```

Comment:
You mentioned [here][link] that:
> My project (the Flatcar sponsored project) doesn't allow neither baremetal_0 nor c3.small.x86

Is that a particular setting of the project? I mean, I'd like to use as default something that will work for most projects. I guess this was something on that project and not some generic packet issue?

Just to be sure that this instance type example makes sense as a quickstart default.

[link]: https://github.com/kinvolk/lokomotive/pull/736#issuecomment-665633971

Reply:
I re-checked this and the issue is that c3.small is only available in Dallas and Sunnyvale (at least for the Flatcar Infra project) and I was deploying the cluster in Amsterdam. So yeah, I think we really need to have this here.

Reply:
Hm, ok, doesn't seem like the best default but I understand the situation will change and is already the default when no param is specified. So, LGTM :)